.cirrus.yml

```diff
@@ -135,15 +136,18 @@ vendor_task:
 # Confirm cross-compile ALL architectures on a Mac OS-X VM.
 cross_build_task:
     name: "Cross Compile"
+    gce_instance:
+        cpu: 8
+        memory: "24G"
     alias: cross_build
     only_if: >-
         $CIRRUS_CHANGE_TITLE !=~ '.*CI:DOCS.*'
     env:
         HOME: /root
     script:
         - go version
-        - make cross CGO_ENABLED=0
-
+        - make -j cross CGO_ENABLED=0
+        - du -sh `go env GOCACHE`
```

Comment:
I think we can drop the du or do you still need that?

Reply:
I'm concerned about using /tmp for the Go cache, but you're probably right that we don't need to be worrying about that in every task.